Finjector.Web/ClientApp/src/App.test.tsx

```diff
@@ -20,6 +20,69 @@ describe("App", () => {
         screen.getByText("New Chart String from Scratch")
       ).toBeInTheDocument();
     });
+
+    await waitFor(() => {
+      expect(screen.getByText("Finjector")).toBeInTheDocument();
+    });
+    await waitFor(() => {
+      expect(screen.getByText("Chart Strings")).toBeInTheDocument();
+    });
+    await waitFor(() => {
+      expect(screen.getByText("Teams")).toBeInTheDocument();
+    });
+    await waitFor(() => {
+      expect(screen.getByText("Help")).toBeInTheDocument();
+    });
```

Comment:
you can put these all in the same waitFor().  once the first one is found they should all be there.  Waitfor is just used to wait for the page to render/loading states.